draft-ietf-gnap-core-protocol.md

```diff
@@ -1626,12 +1628,19 @@ with other values defined by the [Interaction Finish Methods Registry](#IANA-int
 
 
 If interaction finishing is supported for this client instance and
-request, the AS [returns a nonce](#response-interact-finish) used by the client
+request, the AS will [return a nonce](#response-interact-finish) used by the client
 instance to validate the callback.
-Requests to the callback URI MUST be processed as described in
-{{interaction-finish}}, and the AS MUST require
-presentation of an interaction callback reference as described in
-{{continue-after-interaction}}.
+All interaction finish methods MUST use this nonce to allow the client verify the connection
```

Comment:
I think there is a "to" missing in "allow the client TO verify"

Reply:
```suggestion
All interaction finish methods MUST use this nonce to allow the client to verify the connection
```